spec/unit/yardstick/class_methods/measure_spec.rb

```diff
@@ -8,10 +8,11 @@
 
     it 'delegates to Processor' do
       processor = double('processor')
-      Yardstick::Processor.stub(:new).with(instance_of(Yardstick::Config)) do
-        processor
-      end
-      processor.should_receive(:process)
+      config = instance_of(Yardstick::Config)
+
+      allow(Yardstick::Processor)
+        .to receive(:new).with(config).and_return(processor)
+      expect(processor).to receive(:process)
```

Comment:
Below in spec/unit/yardstick/class_methods/measure_string_spec.rb you have basically the same expression, yet totally different formatting. Assuming the line length isn't too long, I prefer the other formatting.